pkg/memlimit/memlimit.go

```diff
@@ -0,0 +1,41 @@
+package memlimit
+
+import (
+	"os"
+	"runtime/debug"
+	"strconv"
+
+	"github.com/stackrox/rox/pkg/logging"
+)
+
+var log = logging.LoggerForModule()
+
+func SetMemoryLimit() {
```

Comment:
It would be nice to test this function

Reply:
>  if tests in other packages run in parallel with this
Do we run tests in parallel?
And you can set env in a test then run function and then check the limit. Maybe it will even make code more readable to have a function that returns limit and another that run `debug.SetMemoryLimit(l)` no defer, less indent

Reply:
the defer was removed and testing was addressed